sources/nodes/kube.go

```diff
@@ -78,8 +78,15 @@ func (self *kubeNodes) List() (*NodeList, error) {
 			nodeList.Items[Host(node.Name)] = Info{PublicIP: node.Status.HostIP, InternalIP: addrs[0].String()}
 			goodNodes = append(goodNodes, node.Name)
 		} else {
-			glog.Errorf("Skipping host %s since looking up its IP failed - %s", node.Name, err)
-			self.recordNodeError(node.Name)
+			ip := net.ParseIP(node.Name)
```

Comment:
Just so that I understand clearly, the hostname of the machines in the cluster matches the IP? Does the rest of kubernetes work given the resolution issues?
Sorry for being a pain, I am just trying to understand the underlying problem!